proposals/023-cooperative-matrix.md

```diff
@@ -12,6 +12,11 @@ Author: Darren Wihandi
 
 Reviewer: TBD
 
+Update:
```

Comment:
Can you update the reviewer status here?

Also please review the SP numbers and make sure they don't clash with existing proposals.